chainer/distribution.py

```diff
@@ -0,0 +1,420 @@
+import copy
+
+
+class Distribution(object):
+
+    """Interface of Distribution
+
+    `Distribution` is a bass class to treat probability distributions.
+
+    This class provides a mean to perform following operations.
```

Comment:
Wording: some examples:

This class provides the following capabilities.
This class can perform the following operations.